script.js

```diff
@@ -1,12 +1,207 @@
-//You can edit ALL of the code here
-function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+//Variables
+// Div for showing episodes or movies
+const showEpisodseRow = document.querySelector(".row");
+// Search input
+const searchUser = document.querySelector("#searchUser");
+// Show the number of episodes based on search
+const searchNumResult = document.querySelector(".epi-num");
+// Select related to shows
+const selectShows = document.querySelector("#select-shows");
+// Select related to episodes
+const selectEpisodes = document.querySelector("#select-episodes");
+// get all shows
+const allShows = getAllShows();
```

Comment:
This doesn't use fetch right now - can you make it use fetch?

Reply:
I changed my code and used fetch 🙏